shell/platform/android/io/flutter/embedding/engine/FlutterEngine.java

```diff
@@ -316,6 +316,20 @@ public void removeEngineLifecycleListener(@NonNull EngineLifecycleListener liste
     engineLifecycleListeners.remove(listener);
   }
 
+  /**
+   * Notify the Dart VM that the system is experiencing memory pressure, and
+   * extra work should be taken to clean up internal resources and collect
```

Comment:
Let's be more transparent here. Since this happens on backgrounding from onTrimMemory, let's not make the name and comment so ominous. Let's just say it's a good time to clean up.

Reply:
I'd like to keep the name so that it's clearer. This is not API that people should call indiscriminately. And this way it's much easier to trace what it actually does on the C++ side.